tools/syz-linter/linter.go

```diff
@@ -61,14 +61,13 @@ var SyzAnalyzer = &analysis.Analyzer{
 func run(p *analysis.Pass) (interface{}, error) {
 	pass := (*Pass)(p)
 	for _, file := range pass.Files {
+		stmts := make(map[int]bool)
 		ast.Inspect(file, func(n ast.Node) bool {
+			if n == nil {
```

Comment:
These aren't comment stylistic changes.

Reply:
Right. This is a separate commit to the linter code that does these checks ;)

Reply:
Maybe extend the change description to cover it?

Reply:
You mean PR description? These are thrown-away after merge.
I think it's only useful for something that's relevant during review only (e.g. questions to reviewers).
Otherwise a PR description could be a concatenation of descriptions of all included commits (b/c if there is anything useful it should not be only in the PR description, which is throw-away). But in this form, it's also kinda pointless to just copy paste from commits.
I am trying to write commit descriptions. Some are longer, some are shorter. But at least there are 2 commits with 2 separate descriptions and the linter change commit says it's a linter change commit.

Reply:
I've updated the PR description.